site/blueprints/email.yml

```diff
@@ -6,6 +6,7 @@ pages: true
     - newsletter
     - basic
     - variety
+    - breakground
```

Comment:
Make sure we remove breakground template from the main templates. We only want it accessible within the breakground email category (email-breakground.yml).